### ISSUE TITLE
 sap_ha_*: integrated tasks for firewalld local configuration to allow traffic

### ISSUE THREAD

**Issue by sean-freeman** [sap-linuxlab/community.sap_install#168]
Self-contained within the Ansible Role, is a requirement for Ansible Tasks to perform firewalld local configuration to allow traffic for SAP HANA System Replication (HSR).

See:
- https://github.com/sap-linuxlab/community.sap_install/pull/161#discussion_r909554649
- https://github.com/sap-linuxlab/community.sap_install/pull/161#discussion_r909561822

**Comment by sean-freeman** [comment]
@ja9fuchs was this removed during re-write of `sap_ha_pacemaker_cluster` ?

**Comment by ja9fuchs** [comment]
Yes, because we already agreed that firewall setup should be handled through a central place, like through including the firewalld LSR and feeding the ports that the individual role defines for the services the role covers.
